bin/fiwareDeviceSimulatorCLI

```diff
@@ -37,7 +37,20 @@ var path = require('path');
 var request = require('request');
```

Comment:
New entry should be included in CNR file. Maybe something like this:

```
- Fix: per-directory .env files
```

Reply:
Added